ecms_base/features/custom/ecms_event/config/install/core.base_field_override.node.event.status.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - node.type.event
+id: node.event.status
+field_name: status
+entity_type: node
+bundle: event
+label: Published
+description: ''
+required: false
+translatable: true
```

Comment:
Status should be translatable false

Reply:
When I change the published field for the content type to not-translatable, I cannot create a draft of a new translation. For example, if an English translation is published and I go to create a draft of a Spanish translation, I get an error.

Reply:
Hm - well ok!